airflow/www/templates/airflow/graph.html

```diff
@@ -103,15 +113,16 @@
     var initialStrokeWidth = '3px';
     var highlightStrokeWidth = '5px';
 
-    var nodes = {{ nodes|safe }};
-    var edges = {{ edges|safe }};
-    var execution_date = "{{ execution_date }}";
-    var arrange = "{{ arrange }}";
+    var nodes = JSON.parse(getMetaValue('nodes'));
+    var edges = JSON.parse(getMetaValue('edges'));
+    var execution_date = getMetaValue('execution-date');
+    var arrange = getMetaValue('arrange');
+    var tasks = JSON.parse(getMetaValue('tasks'));
+    var task_instances =  JSON.parse(getMetaValue('task-instances'));
```

Comment:
I see that this was the original variable name, but if we're cleaning up the js, maybe we can use camelCase consistently.

Reply:
I would like to introduce it in a separate PR. I have a PR in queue, which introduce linting for all JS (*.js, *.html). This allows me to deal with the problem globally instead of making changes only locally. Solving the problem locally will not be permanent, so it should only be used when there is no other possibility.